pyaerocom/io/read_ebas.py

```diff
@@ -17,31 +17,51 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 # MA 02110-1301, USA
-from geonum.atmosphere import p0, T0_STD
-import os, re
 import fnmatch
-import numpy as np
+import os
+import re
 from collections import OrderedDict as od
+from warnings import catch_warnings, filterwarnings
+
+import numpy as np
+from tqdm import tqdm
+
+with catch_warnings():
+    filterwarnings("ignore")
+    from geonum.atmosphere import T0_STD, p0
```

Comment:
Is that the basemap warning? Will be removed in geonum in upcoming release of v1.5.0.